slsDetectorGui/src/qTabPlot.cpp

```diff
@@ -389,6 +389,48 @@ void qTabPlot::SetTitles() {
             SLOT(SetTitles()));
 }
 
+void qTabPlot::isXMinModified() {
```

Comment:
These additional (in between) slots are necessary because all the chkXMin  (check box) and dispXMin and dispXMax (text fields) calls SetXRange... so had to to add IsXMinModified() when dispXMin editingFinished() gets called.. before calling SetXRange